README.md

```diff
@@ -1,6 +1,8 @@
 # RepoSense
 [![Build Status](https://travis-ci.org/reposense/RepoSense.svg?branch=master)](https://travis-ci.org/reposense/RepoSense)
 [![Build status](https://ci.appveyor.com/api/projects/status/gsbkj5qby3pjd6nw/branch/master?svg=true)](https://ci.appveyor.com/project/eugenepeh/reposense/branch/master)
+[![Codacy Badge](https://api.codacy.com/project/badge/Grade/08a3527378464ed4a5ad62e27f590d6a)](https://www.codacy.com/app/eugenepeh/RepoSense?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=reposense/RepoSense&amp;utm_campaign=Badge_Grade)
```

Comment:
Can we set the badge to be from RepoSense project instead? (Appveyor badge seems to be from your side as well)

Reply:
Done. For Appveyor, it seems like they do not have an organization system. Hence, I can't create an organization and transfer the project to it.